src/modules/person/api/person.controller.ts

```diff
@@ -132,16 +162,21 @@ export class PersonController {
     @ApiBadRequestResponse({ description: 'The personenkontext already exists.' })
     @ApiUnauthorizedResponse({ description: 'Not authorized to create the personenkontext.' })
     @ApiForbiddenResponse({ description: 'Not permitted to create the personenkontext.' })
+    @ApiNotFoundResponse({ description: 'Insufficient permissions to create personenkontext for person.' })
     @ApiInternalServerErrorResponse({ description: 'Internal server error while creating the personenkontext.' })
     public async createPersonenkontext(
         @Param() pathParams: PersonByIdParams,
         @Body() bodyParams: CreatePersonenkontextBodyParams,
+        @Permissions() permissions: PersonPermissions,
     ): Promise<PersonenkontextResponse> {
         const personenkontextDto: CreatePersonenkontextDto = this.mapper.map(
             bodyParams,
             CreatePersonenkontextBodyParams,
             CreatePersonenkontextDto,
         );
+        //check that logged-in user is allowed to update person
+        await this.personUc.getPersonIfAllowed(pathParams.personId, permissions);
```

Comment:
`personUc.getPersonIfAllowed` just returns an error if the user is not allowed to access the person, your discarding the result here